app/src/org/commcare/android/view/SquareButton.java

```diff
@@ -27,7 +27,12 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
         int w = MeasureSpec.getSize(widthMeasureSpec);
         int h = MeasureSpec.getSize(heightMeasureSpec);
```

Comment:
You can avoid the comment by naming 'w' and 'h' to 'canvasWidth' and 'canvasHeight'